config/routes.rb

```diff
@@ -19,4 +22,7 @@
 
     end
   end
+  get 'login', to: 'login#new'
+  post 'login', to: 'login#create'
+  delete 'logout', to: 'login#destroy'
```

Comment:
It's odd to me to have both OpenStax::Accounts and a separate login controller mounted since the OpenStax::Accounts gem provides a login controller.

Reply:
I had no idea the gem provided that and there is no documentation, so I built a simple UI.